dfs/restore_test.go

```diff
@@ -54,7 +54,7 @@ func (s *DFSTestSuite) TestRestore_LoadImages(c *C) {
 	err := tarfile.WriteHeader(&tar.Header{Name: DockerImagesFile, Size: 0})
 	c.Assert(err, IsNil)
 	tarfile.Close()
-	s.docker.On("LoadImage", mock.AnythingOfType("*tar.Reader")).Return(nil)
+	s.docker.On("LoadImage", mock.Anything).Return(nil)
```

Comment:
The reader was wrapped with a TeeReader to so I could count the bytes.  The TeeReader type (io.teeReader) is not exported so I couldn't change it to that.  I tried the interface io.Reader, but that didn't work either with AnythingOfType.  I changed it to use Anything instead.  When its used, the value gets casted in the next line which would through a panic if something was wrong, so I think it will still catch issues that appear.